pkgs/applications/science/electronics/kicad/base.nix

```diff
@@ -165,6 +173,14 @@ stdenv.mkDerivation rec {
   ++ optional (debug) valgrind
   ;
 
+  postInstall = optionalString stdenv.isDarwin ''
```

Comment:
i would do the `cp` and the `makeWrapper` in the main package
which wraps and passes through the built products from the `kicad.base` package

basically, the stuff in base.nix is intended to be just the building and testing of the build
with the stuff in default.nix being just the packaging to make it usable